scipy/stats/_distn_infrastructure.py

```diff
@@ -3177,7 +3182,11 @@ def expect(self, func=None, args=(), loc=0, lb=None, ub=None,
         slowly, the accuracy of the result may be rather low. For instance, for
         ``zipf(4)``, accuracy for mean, variance in example is only 1e-5.
         increasing `maxcount` and/or `chunksize` may improve the result, but may
-        also make zipf very slow.
+        also make zipf very slow. Neither this function nor `integrate.quad` can 
```

Comment:
integrate.quad is not used in the discrete case.
Almost all discrete distribution are for non-negative integers, so there is (most likely) no case where mean is nan.
I think we weaken this, there is already the warning note about heavy tailed distribution, e.g.

`If the expected value is `np.nan` or `np.inf`, then the `expect` method  will return a numeric approximation that might not match either `np.nan` or  `np.inf`.`

example, I didn't check whether it makes sense
>>> stats.zipf.mean(1.1)
inf
>>> stats.zipf(1.1).expect()
48.27458276042281
```

Reply:
fair enough, plus now looking at that string I added it looks like I cannot spell "intergal" correctly.  👍 

Do I make the change (removing the "Neither this function..." part) and re-push to my remote branch?

Reply:
just removed the 2 sentences and update local via `git commit --amend`  and pushed to my remote via `git push --force origin scipy_issue_9259` 

That should address the phrasing. Also I think the travis CI build fail is inherited from the mainline. The build fail has nothing to do w/the changes I made.